### PR TITLE
[render tests] Add streets-v11 test for the tile map mode

### PR DESCRIPTION
**Pull request by pozdnyakov**

### CODE REVIEW

File: test/ignores.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "render-tests/fill-pattern/update-feature-state": "https://github.com/mapbox/mapbox-gl-js/issues/7207",
   "render-tests/text-size/zero": "https://github.com/mapbox/mapbox-gl-js/issues/9161",
   "render-tests/text-variable-anchor/left-top-right-buttom-offset-tile-map-mode": "skip - mapbox-gl-js does not need to render tiles",
+  "render-tests/tile-mode": "skip - mapbox-gl-js does not need to render tiles",
```
Comment (tmpsantos) [inline test/ignores.json]: I would rephrase it as doesn't support "tile mode". :-)

File: test/ignores.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
   "render-tests/fill-pattern/update-feature-state": "https://github.com/mapbox/mapbox-gl-js/issues/7207",
   "render-tests/text-size/zero": "https://github.com/mapbox/mapbox-gl-js/issues/9161",
   "render-tests/text-variable-anchor/left-top-right-buttom-offset-tile-map-mode": "skip - mapbox-gl-js does not need to render tiles",
+  "render-tests/tile-mode/streets-v11": "skip - mapbox-gl-js does not need to render tiles",
```
Comment (asheemmamoowala) [inline test/ignores.json]:
```suggestion
  "render-tests/tile-mode/streets-v11": "skip - mapbox-gl-js does not support tile-mode",
```
